custom/samveg/case_importer/validators.py

```diff
@@ -0,0 +1,12 @@
+class BaseValidator:
+    @classmethod
+    def run(cls, *args, **kwargs):
+        raise NotImplementedError
+
+
+class MandatoryColumnsValidator(BaseValidator):
```

Comment:
Ya. Those PRs are still up but the project needs this done this week itself.

On that, next step is for me to right specs and then share with USH and then it would be considered further. I am guessing once this is introduced in DD, we can remove code from here and use DD instead or may be just keep these. Will depend on how things are once we get to that point.